datalad/support/sshconnector.py

```diff
@@ -158,6 +162,40 @@ def open(self):
     def close(self):
         raise NotImplementedError
 
+    @property
+    def ssh_command(self):
+        """determine which ssh client should be used.
+
+        On Windows OpenSSH for Windows is required. It is searched in its
+        default location. This can be overridden by setting the datalad
+        config variable 'datalad.windows.ssh.client' to the path of the
+        system ssh client.
+        """
+        if self._ssh_command:
+            return self._ssh_command
+
+        # If on windows, enforce use of windows provided ssh client. This is
+        # done because other ssh clients, for example the git-bundled
+        # ssh-client, might not work on windows.
+        if on_windows:
+            from datalad import cfg
+
+            cmd = cfg.obtain(self.cfg_datalad_windows_ssh_client)
+            if not Path(cmd).exists():
+                lgr.error(
+                    f"SSH client ({cmd}) was not found.\nPlease "
+                    f"install OpenSSH for Windows. If you have installed "
+                    f"OpenSSH for Windows in another location than the default "
+                    f"location, please set the datalad configuration key "
+                    f"'{self.cfg_datalad_windows_ssh_client}' to the "
+                    f"location of the installed 'ssh.exe' file.")
+                raise RuntimeError("ssh-executable not found")
+            self._ssh_command = cmd
+        else:
+            self._ssh_command = "ssh"
```

Comment:
This is now duplicating the logic in the cfg default setup. Moreover, the switching and redeclaration of a default should not be platform-specific here, for the same reason.

Reply:
Thx, I overlooked that in the torrent of activities and changes